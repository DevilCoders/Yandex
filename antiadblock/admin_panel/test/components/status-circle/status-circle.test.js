import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

import StatusCircle from 'app/components/status-circle/status-circle';

describe('status-circle component', () => {
    test('active status', () => {
        const wrapper = shallow(<StatusCircle status='active' />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });

    test('empty status', () => {
        const wrapper = shallow(<StatusCircle />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
