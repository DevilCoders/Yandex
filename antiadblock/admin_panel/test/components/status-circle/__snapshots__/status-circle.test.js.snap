// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`status-circle component active status 1`] = `
<div
  className="status-circle status-circle_small status-circle_active"
/>
`;

exports[`status-circle component empty status 1`] = `
<div
  className="status-circle status-circle_small "
/>
`;
